frontend/javascripts/oxalis/store.js

```diff
@@ -467,6 +467,7 @@ export type OxalisState = {|
   +viewModeData: ViewModeData,
   +activeUser: ?APIUser,
   +uiInformation: UiInformation,
+  +isosurfaces: Object,
```

Comment:
From a pure JavaScript-perspective, it is true that this is simply an `Object`. However, we can (and should) provide more information to flow so that the static validation is more powerful. See https://flow.org/en/docs/types/objects/#toc-objects-as-maps. So, this should be something like:

```js
type IsosurfaceInfo = {
  ..
}
...

type ... = {
  ...
  +isosurfaces: {[segmentId: string]: IsosurfaceInfo}
  ...
}
```

Reply:
Yes, that's definitely better, thank you.